web/testing/issue6005/vowel_hater/source/vowel_hater.kmn

```diff
@@ -0,0 +1,96 @@
+﻿c vowel_hater generated from template at 2022-04-11 11:51:48
+c with name "Vowel Hater"
+c
+c This test-oriented keyboard "hates" words with more than two consecutive vowels.
+c In order to test within-word applications of non-final rule matching with final rule non-matching,
+c Vowels are NOT emitted by keyboard rule; only by 'default' behavior.
+c
+c For test coverage, also really likes duplicating 'y' and 'Y'.
+
+store(&VERSION) '10.0'
+store(&NAME) 'Vowel Hater'
+store(&COPYRIGHT) '© SIL Int' U+0027 'l'
+store(&KEYBOARDVERSION) '1.0'
+store(&TARGETS) 'any'
+store(&VISUALKEYBOARD) 'vowel_hater.kvks'
+store(&LAYOUTFILE) 'vowel_hater.keyman-touch-layout'
+
+store(vowels) 'aeiouAEIOU' c Doesn't hate 'y' or 'Y'.
```

Comment:
```suggestion
store(vowels)     'aeiouAEIOU' c Doesn't hate 'y' or 'Y'.
store(vowel_keys) [K_A] [K_E] [K_I] [K_O] [K_U] \
                  [SHIFT K_A] [SHIFT K_E] [SHIFT K_I] [SHIFT K_O] [SHIFT K_U] 
store(cons)       'bcdfghjklmn' \
                  'pqrstvwxz' \
                  'BCDFGHJKLMN' \
                  'PQRSTVWXZ'
store(consKeys)   [SHIFT K_B] [SHIFT K_C] [SHIFT K_D] [SHIFT K_F] [SHIFT K_G] [SHIFT K_H] \
                    [SHIFT K_J] [SHIFT K_K] [SHIFT K_L] [SHIFT K_M] [SHIFT K_N] \
                  [SHIFT K_P] [SHIFT K_Q] [SHIFT K_R] [SHIFT K_S] [SHIFT K_T] [SHIFT K_V] \
                    [SHIFT K_W] [SHIFT K_X] [SHIFT K_Z]
```

Or this has identical behaviour because the chars are translated to vkeys:

```
store(vowels)     'aeiouAEIOU' c Doesn't hate 'y' or 'Y'.
store(vowel_keys) 'aeiouAEIOU'

store(cons)       'bcdfghjklmn' \
                  'pqrstvwxz' \
                  'BCDFGHJKLMN' \
                  'PQRSTVWXZ'
store(cons_keys)  'bcdfghjklmn' \
                  'pqrstvwxz' \
                  'BCDFGHJKLMN' \
                  'PQRSTVWXZ'
```